scripts/build_docs.py

```diff
@@ -0,0 +1,130 @@
+from typing import Dict
+import os
+from pathlib import Path
+from subprocess import check_output
+import argparse
+
+from ruamel.yaml import YAML
+
+
+exclude_files = [".DS_Store", "__init__.py", "__init__.pyc", "README.md", "version.py", "run.py"]
+
+
+def render_docs(relative_src_path: str, src_file: str, to_file: str, modifier="++") -> None:
```

Comment:
The method name makes me think this should operate on more than one file.  You have `build_docs_for_file` below, but just `render_docs` here.  Maybe `render_file` instead?